scipy/stats/tests/test_multivariate.py

```diff
@@ -515,8 +515,13 @@ def test_bad_input(self):
         assert_raises(ValueError, matrix_normal, M, V, V)
         assert_raises(ValueError, matrix_normal, M.T, U, V)
 
-        # Singular covariance
         e = np.linalg.LinAlgError
+        # Singular covariance for the rvs method of a non-frozen instance
+        assert_raises(e, matrix_normal.rvs,
+                      M, U, np.ones((num_cols, num_cols)))
+        assert_raises(e, matrix_normal.rvs,
+                      M, np.ones((num_rows, num_rows)), V)
+        # Singular covariance for a frozen instance
```

Comment:
I prefer to keep PRs focused, but I guess this is fine since it's simple. Two other things that would be improvements:

- matching the error message
- if the array of ones were stored, these could fit on the same line

But let's merge as-is.